Makefile

```diff
@@ -80,6 +80,9 @@ logs.%: ## Shows the logs of the given docker service. Example: make logs.datast
 restart.%: ## Restarts the given docker service. Example: make restart.datastore
 	docker compose restart $*
 
+run_tests.lints: ## Runs the linter tests
+	docker compose run --no-deps --entrypoint "python -m scripts.linters.pre_commit_linter $(flags)" dev-server
```

Comment:
@U8NWXD I need to discuss about how we will be handling flags for the local tests wrapped up using make commands.
on taking reference from nodejs repo --  refer [here](https://github.com/nodejs/node/blob/main/Makefile#L319), the plan is to pass the flags as a string, and then internally I will append this string within the python command we are executing.
The make commands will look like --
`make run_tests.lints` (normal command)
`make run_tests.lints flags="--path Makefile"` (adding flag as a string)

how we handle 2 flags at a time ---
`make run_tests.backend flags="--verbose --generate_coverage_report"`


and @gp201 also agrees on this. whats your opinion @U8NWXD

Reply:
@Shivkant-Chauhan Can't we do it similarly to how we are handling flags for test targets?

Reply:
"test targets"
I think you are discussing about what you added for the backend tests initially in the Makefile. please take a look to [this](https://docs.google.com/document/d/1pvHjw-hFacNIr6z7oDaZ6a2atf0fvbaggpPbRhPkqqo/edit?disco=AAAA2sB_PEo), we have several issues while doing that because bash understands the `--` as the make command args/option

Reply:
Hey @DubeySandeep @U8NWXD, upon discussion with @seanlip, he is okay with passing the flags as a string to the python command using `PYTHON_ARGS` as the argument for the make command. This will be the same for all the local tests that have flags or optional args for the python command.
We can change it to either with separate args for each flag that the python command supports (that @U8NWXD earlier suggested) post-GSoC ends, after discussing it properly according to the requirements.